provider/oci/storage.go

```diff
@@ -5,16 +5,55 @@ package oci
 
 import (
 	"github.com/juju/errors"
+	"github.com/juju/utils/clock"
 
+	providerCommon "github.com/juju/juju/provider/oci/common"
 	"github.com/juju/juju/storage"
 )
 
-type storageProvider struct{}
+type poolType string
+
+const (
+	ociStorageProviderType = storage.ProviderType("oci")
+
+	// maxVolumeSizeInGB represents the maximum size in GiB for
+	// a single volume. For more information please see:
+	// https://docs.oracle.com/cloud/latest/stcomputecs/STCSA/op-storage-volume--post.html#request
+	maxVolumeSizeInGB = 16000
+	// minVolumeSizeInGB represents the minimum size in GiB for
+	// a single volume. For more information please see:
+	// https://docs.oracle.com/cloud/latest/stcomputecs/STCSA/op-storage-volume--post.html#request
+	minVolumeSizeInGB = 50
```

Comment:
I realize this is a different storage, but how does this compare with the ones we use for root disk? Can these values be shared, should they be caveated so that it is clear which one is used when?
minRootVolumeSize, etc?
And then its a little odd that one is in MB and one is in GB